bundle/manifests/rhdh.redhat.com_orchestrators.yaml

```diff
@@ -351,20 +326,16 @@ spec:
                         default: redhat-operators
                         description: Source captures the name of the catalog source
                         type: string
-                      sourceName:
-                        default: redhat-operators
-                        description: SourceName captures the name of the catalog source
-                        type: string
                       startingCSV:
                         default: ""
                         description: The initial version of the operator
                         type: string
-                      targetNamespace:
-                        default: rhdh-operator
-                        description: The target namespace for the backstage CR in
-                          which RHDH instance is created
-                        type: string
                     type: object
+                  targetNamespace:
+                    default: rhdh-operator
+                    description: The target namespace for the backstage CR in which
```

Comment:
This field `targetNamespace` should be storing the target namespace of the RHDH operator subscription.
For the default namespace for the Backstage CR, we should pick a different one, perhaps the `orchestrator` or require the use to specify if the RHDH is disabled.